lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php

```diff
@@ -626,7 +636,20 @@ private function joinColumnToArray(Mapping\JoinColumn $joinColumn)
      * @param string $fieldName
      * @param Mapping\Column $column
      *
-     * @return array
+     * @return mixed[]
+     *
+     * @psalm-return array{
+     *                   fieldName: string,
+     *                   type: mixed,
+     *                   scale: int,
+     *                   length: int,
+     *                   unique: bool,
+     *                   nullable: bool,
+     *                   precision: int,
+     *                   options?: array,
```

Comment:
Array of what?

Reply:
Of "both"! Ya... if you use `['foo' => true]` or `['foo']`, it'll lead to the same result and will become `FOO` on column definition.

Reply:
Let's use `mixed[]` here I guess :sweat_smile: